e2e/src/utils.ts

```diff
@@ -553,6 +554,16 @@ export const utils = {
     await immichCli(['login', app, `${key.secret}`]);
     return key.secret;
   },
+
+  scan: async (accessToken: string, id: string, wait = false) => {
```

Comment:
The reason why we had the scan method in the library spec file itself is because no other test needs that functionality. It's probably fine either way though

Reply:
I debated back and forth on this and eventually put it in the utils because I see it being used in other e2e tests soon :)